packages/server/graphql/mutations/setMeetingSettings.ts

```diff
@@ -44,8 +45,9 @@ const setMeetingSettings = {
     if (!settings) {
       return standardError(new Error('Settings not found'), {userId: viewerId})
     }
-    const {teamId} = settings
+    const {teamId, meetingType} = settings
 
+    const meetingSettings = {}
```

Comment:
-1 You can type this here `const meetingSettings = {} as MeetingSettings` then you don't need to do the `Object.assign` trick to bypass typings.